content/doc/developer/plugin-development/choosing-jenkins-baseline.adoc

```diff
@@ -5,24 +5,37 @@ title: Choosing a Jenkins version to build against
 
 Your plugin's POM controls which version of Jenkins it is building/testing against.
 
-## Choosing a version
+== Choosing a version
 
 You need to balance compatibility and features:
 
-* *Keeping the Jenkins version your plugin builds against low* will allow more users to install and use your plugin. 
+* *Keeping the Jenkins version your plugin builds against low* will allow more users to install and use your plugin.
 In particular, the LTS Release Line is based on slightly older releases to provide a more stable experience for conservative users.
-* *Building against recent Jenkins versions* allows you to use recently added core features and API from your plugin. 
-You will also use that newer version for `mvn hpi:run`, so it may be easier to test your plugin with newer Jenkins releases. 
+* *Building against recent Jenkins versions* allows you to use recently added core features and API from your plugin.
+You will also use that newer version for `mvn hpi:run`, so it may be easier to test your plugin with newer Jenkins releases.
 Additionally, since features are sometimes moved from core into plugins, depending on a more recent Jenkins version will make your plugin's dependencies on what used to be core features explicit, otherwise your plugin will not work.
-* *Use at least the minimum version supported by the update center*, the supported update center versions can be found at link:https://updates.jenkins.io[updates.jenkins.io], at the time of writing (2020-03) this is 2.164.x.
+* *Use at least the minimum version supported by the update center*, the supported update center versions can be found at link:https://updates.jenkins.io[updates.jenkins.io], at the time of writing (August 2020) this is 2.176.x.
 
-There are link:https://stats.jenkins.io/pluginversions/[statistics of core versions by plugin] that can help you decide.
+There are link:https://stats.jenkins.io/pluginversions/[statistics of core versions by plugin] that can help you decide. These are updated monthly.
 
-These are updated monthly.
+
+== Suggested minimum version
 
 General tip: When upgrading, choose the link:/changelog-stable/[newest LTS version] that doesn't alienate the majority of users of your latest few releases (by requiring a newer Jenkins than they have).
 
-## Changing the minimum required version
+If you do not have special requirements then a good idea is to support the latest three
+LTS minor versions. This helps to ensure that teams that are slower at adapting new Jenkins versions 
+will not be forced to update their Jenkins too frequently. Additionally, other plugins that 
```

Comment:
I'm not understanding your point of disagreement, @daniel-beck, can you explain further?

I think this paragraph is trying to guide plugin maintainers to choose a minimum required Jenkins version that includes a significant portion of their users.  I don't think it is trying to claim that the Jenkins project or the Jenkins security team provide support for Jenkins core releases of anything other than the current LTS release line.

As a specific example, when choosing the minimum Jenkins version that the git plugin would support, I checked the installation statistics for the git plugin and decided based on installation patterns that Jenkins 2.204.1 was a good choice for the minimum required Jenkins version because that would not force a Jenkins version upgrade for over 60% of the users that were running a recent release of the plugin.

I don't think that you're recommending that plugins prefer to require the most recent LTS release line as their minimum Jenkins version, but maybe I've misunderstood your intent?

Reply:
Depends on security content in recent releases; but to me it sends mixed signals when we recommend plugin maintainers choose a baseline that is known to have security vulnerabilities specifically to allow administrators to postpone updating.

Reply:
The baseline is only a compile dependency, so I do not understand your point. I think everybody developing Jenkins plugins recommends the latest LTS when someone asks which version to install. 

Or do you mean that we should remove the reasoning "This helps to ensure that teams that are slower at adapting new Jenkins versions will not be forced to update their Jenkins too frequently."

Reply:
> Or do you mean that we should remove the reasoning "This helps to ensure that teams that are slower at adapting new Jenkins versions will not be forced to update their Jenkins too frequently."

Yes, the justification we give here undermines our general message to keep Jenkins up to date and especially to take security issues seriously. I don't think we've gone two consecutive LTS baselines in recent years without at least one security update.